src/main/java/com/origamisoftware/teach/agile/junit/SubscriptionPeriod.java

```diff
@@ -43,17 +58,25 @@ public Date getEndDate() {
      * @return the total Days in the subscription
      */
     public int getTotalDays() {
-        // todo implement method
-        return 0;
+        return (int) ((endDate.getTime() - startDate.getTime())/MSECONDS_PER_DAY);
     }
 
     /**
      * @return the total months on the subscription
      */
     public int getTotalMonths() {
-        // todo implement method
+        Calendar stop = Calendar.getInstance();
+        stop.setTime(endDate);
+        Calendar start = Calendar.getInstance();
+        start.setTime(startDate);
+        //pulled code from test(in Agile project). Hope that's okay.
```

Comment:
you mean you used the test code to help right the real method? That's ok. Wait until you read the test driven development material.